.claude/scripts/keyserve.py

```
@@ -0,0 +1,88 @@
+import os
+import re
+import secrets
+import sys
+import threading
+
+ALLOWED = {'HEYGEN_API_KEY', 'IDEOGRAM_API_KEY'}
+TIMEOUT = 180
+
+def main() -> int:
+    if len(sys.argv) != 3:
+        print(__doc__); return 2
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)